lib/services/networkManagement2/lib/models/expressRouteCircuitsStatsListResult.js

```diff
@@ -25,8 +27,6 @@ var util = require('util');
 function ExpressRouteCircuitsStatsListResult() {
 }
 
-util.inherits(ExpressRouteCircuitsStatsListResult, Array);
-
```

Comment:
@deepakswifty - This change means that "x-ms-pageable-operation" extension was removed from the operation that provided expressRouteCircuitsStatsListResult as the response.

Even if the operation is not pageable, it is good to have the extension.
You can explicilty specify the nextLink to null in the extension. That will not generate the nextLink but till still provide a user friendly clientside API. 

In node.js the user will directly get the array referenced by value property a.k.a value is flattened and in C# the user will get an IEnumerable<ExpressRouteCircuitStats>.

Storage does the same thing for listing storage accounts